barretenberg/cpp/src/barretenberg/vm/tests/avm_execution.test.cpp

```diff
@@ -2135,73 +2135,73 @@ TEST_F(AvmExecutionTests, kernelOutputHashExistsOpcodes)
-                         "00000100"            // val i
-                         "00000012" +          // dst_offset 0
-                         to_hex(OpCode::SET) + // opcode SET for value stored in args offset
-                         "00"                  // Indirect flag
-                         "03"                  // U32
-                         "00000012"            // val i
-                         "00000001";
-    // ret offset preamble
-    bytecode_preamble += to_hex(OpCode::SET) + // opcode SET for ret offset indirect
-                         "00"                  // Indirect flag
-                         "03"                  // U32
-                         "00000008"            // val i
-                         "00000004" +          // dst_offset 0
-                         to_hex(OpCode::SET) + // opcode SET for value stored in ret offset
-                         "00"                  // Indirect flag
-                         "03"                  // U32
-                         "00000002"            // val i
-                         "00000007";
-    std::string bytecode_hex = bytecode_preamble // SET gas, addr, args size, ret offset, success, function selector
-                               + to_hex(OpCode::CALL) +   // opcode CALL
-                               "15"                       // Indirect flag
-                               "00000000"                 // gas offset
-                               "00000001"                 // addr offset
-                               "00000002"                 // args offset
-                               "00000003"                 // args size offset
-                               "00000004"                 // ret offset
-                               "00000007"                 // ret size
-                               "0000000a"                 // success offset
-                               "00000006"                 // function_selector_offset
-                               + to_hex(OpCode::RETURN) + // opcode RETURN
-                               "00"                       // Indirect flag
-                               "00000008"                 // ret offset 8
-                               "00000003";                // ret size 3
-
-    auto bytecode = hex_to_bytes(bytecode_hex);
-    auto instructions = Deserialization::parse(bytecode);
-
-    std::vector<FF> calldata = {};
-    std::vector<FF> returndata = {};
-
-    // Generate Hint for call operation
-    auto execution_hints = ExecutionHints().with_externalcall_hints(
-        { { .success = 1, .return_data = { 9, 8 }, .l2_gas_used = 0, .da_gas_used = 0 } });
-
-    auto trace = Execution::gen_trace(instructions, returndata, calldata, public_inputs_vec, execution_hints);
-    EXPECT_EQ(returndata, std::vector<FF>({ 9, 8, 1 })); // The 1 represents the success
-
-    validate_trace(std::move(trace));
-}
+// TEST_F(AvmExecutionTests, opCallOpcodes)
```

Comment:
I guess this needs updating in a follow-up?

Reply:
Yep, the tags need updated